mavros_msgs/msg/RPM.msg

```diff
@@ -0,0 +1,4 @@
+# RPM message
+
+std_msgs/Header header
+float32[] rpm
```

Comment:
RPM msg just has two fields for RPM. So why not a fixed size array?

Reply:
I would also advise you to add a timestamp field to it on the Mavlink msg (proposing a PR for it). It is much more accurate to have the time that data was generated than having the time that you receive the msg propagated. Though for ground robots it's not really relevant, at least we could have timesync between the onboard companion and the autopilot.

Reply:
Timestamping is a good idea - I just didn't know the messages are allowed to be modified for compatibility reasons. If yes, then of course I will add it.

Reply:
> Timestamping is a good idea - I just didn't know the messages are allowed to be modified for compatibility reasons. If yes, then of course I will add it.

Just create a PR against Mavlink repo with the new field.

Reply:
Btw I see you created a PR for Mavlink to add the timestamp: why not extend it to already support an array instead of having `rpm1` and `rpm2` fields?